Server/JokesOnYou.Web.Api/Controllers/SavedJokeController.cs

```diff
@@ -11,5 +20,19 @@ public SavedJokeController(ISavedJokeService savedJokeService)
         {
             _savedJokeService = savedJokeService;
         }
+
+        [HttpPost("{id}")]
+        public async Task<ActionResult> ToggleSavedJoke(int id)//id here is jokeid
```

Comment:
Maybe Cal it jokeId if its unclear by reading the code.
Better to leave the comments out and let the code speak for itself ?